src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -276,7 +277,10 @@ CHIP_ERROR Hash_SHA256(const unsigned char * data, const size_t data_length, uns
     return error;
 }
 
-Hash_SHA256_stream::Hash_SHA256_stream(void) {}
+Hash_SHA256_stream::Hash_SHA256_stream(void)
+{
+    static_assert(sizeof(mContext) >= sizeof(SHA256_CTX), "Need more memory to store SHA256 context");
```

Comment:
Minor nit; however, if this file has no other C++11 dependencies, recommend using the portable static assertion in nlassert.h.